scripts/shutdown-ocf-contributions.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * A script to cancel all recurring contributions for OCF.
+ */
+
+import '../server/env';
+
+import { QueryTypes } from 'sequelize';
+
+import FEATURE from '../server/constants/feature';
+import OrderStatuses from '../server/constants/order-status';
+import { defaultHostCollective } from '../server/lib/utils';
+import { Collective, sequelize } from '../server/models';
+
+const MESSAGE_FOR_CONTRIBUTORS = `The Fiscal Host of the Collective, Open Collective Foundation, is shutting down its financial operations. We are therefore pausing all recurring financial contributions until Collectives find a new Fiscal Host to operate under.
+<br><br>
+Some helpful resources to learn more about the situation:
+<ul>
+  <li><a href="https://blog.opencollective.com/open-collective-official-statement-ocf-dissolution/">Open Collective Official Statement</a></li>
+  <li><a href="https://opencollective.com/foundation/updates/announcement-we-are-dissolving-open-collective-foundation-at-the-end-of-this-year">Open Collective Foundation's announcement</a></li>
+</ul>
+`;
+
+export const main = async ({ overrideDryRun = false } = {}) => {
+  if (process.env.DRY_RUN !== 'false' && !overrideDryRun) {
+    console.error('DRY_RUN is not set to false, exiting');
+    return;
+  }
+
+  // Disable contributions
+  const ocf = await Collective.findByPk(defaultHostCollective('foundation').CollectiveId);
+  await ocf.update({
+    data: {
+      ...ocf.data,
+      features: {
+        ...ocf.data?.features,
+        [FEATURE.RECEIVE_FINANCIAL_CONTRIBUTIONS]: false,
+      },
+    },
+  });
+
+  // Cancel all existing contributions
+  // The query below is a copy-paste from `Orders.stopActiveSubscriptions`, adapted to run on the entire Host
+  await sequelize.query(
+    `
+      UPDATE "Orders"
+      SET
+        status = :newStatus,
+        "updatedAt" = NOW(),
+        "data" = COALESCE("data", '{}'::JSONB) || JSONB_BUILD_OBJECT(
+          'messageForContributors', :messageForContributors,
+          'messageSource', :messageSource,
+          'needsAsyncDeactivation', TRUE
```

Comment:
a flag could help to track stats

```suggestion
          'deactivatedForOCFShutdown', TRUE
```